backend/src/neighbourhood.rs

```diff
@@ -87,17 +87,17 @@ impl NeighbourhoodDefinition {
         let props = feature
             .properties
             .as_mut()
-            .expect("NeighbourhoodBoundary always has properties");
+            .expect("NeighbourhoodDefinition always has properties");
```

Comment:
No behavioral changes in this file - just some things which didn't get updated after a rename.